crates/nu-command/src/platform/input.rs

```diff
@@ -83,8 +86,36 @@ impl Command for Input {
                 let _ = std::io::stdout().flush();
             }
 
-            // Just read a normal line of text
             let mut buf = String::new();
+
+            if suppress_output {
+                let _ = crossterm::terminal::enable_raw_mode();
+                loop {
+                    let event = crossterm::event::read()?;
+                    if let Event::Key(k) = event {
+                        match k.code {
+                            // TODO: maintain keycode parity with existing command
+                            KeyCode::Char(_) if k.modifiers != KeyModifiers::NONE => continue,
+                            KeyCode::Char(c) => buf.push(c),
+                            KeyCode::Backspace => {
+                                let _ = buf.pop();
+                            }
+                            KeyCode::Enter => break,
+                            _ => continue,
+                        }
```

Comment:
With no arguments, `input` just reads a line from `stdin` and captures keycodes like `Ctrl-C` and `Esc`. Assuming we want to collect the same keycodes with `-s` as without, what should I reference to properly emulate how the shell handles keys for `stdin`?

Reply:
Since no one else responded. Let's just run with what you have and see if it breaks anything downstream. Thanks!